<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>601</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>601</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GameLauncherPatcher</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCommandLinkButton" name="btnPatch">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>200</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Patch</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QToolButton" name="btnBrowseInputDirectory">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtInputDirectoryLocation">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtOutputDirectoryLocation">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="btnBrowseOutputDirectory">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPatchFileName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
